nautobot/core/management/commands/build_ui.py

```diff
@@ -37,6 +37,13 @@ def add_arguments(self, parser):
             default=True,
             help="Do not compile UI.",
         )
+        parser.add_argument(
+            "--allow-ui-build-failure",
+            action="store_true",
+            dest="allow_ui_build_failure",
+            default=True,
+            help="Allow build errors in development (UI may fail to build)",
+        )
```

Comment:
As stated above, allow the UI build to fail and still start Django in development cases.

Reply:
Actually, how is this different from `--no-npm-build`?

Reply:
It attempts to build but allows it to fail.